### DISCUSSION TITLE
basic, basic user

### DISCUSSION THREAD

**Discussion by mingota** [community/community#35620] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello all,
I'm trying to take a stab at using this site to display my writing.
I was taking an Information Technologies course and the professor used this platform instead of the one associated with the school.
I found a site on here whose layout I would like to use.. from what I understand that is welcome here but I'm unsure how to replicate it. and insert my own information within the layout.
So I am predominantly concerned with the presentation of it on the outside. No knowledge of writing code :(
_[removed by moderator]_

I imagine that this is possible but wondering if there is a way for that to be explained simply or should I just abandon this ship entirely.
Thank you

**Comment by sohmc** [comment, marked as answer]
It *really* depends on your level of expertise.  If you truly are a beginner and you have no knowledge of writing code, you're going to have a bad time.  Although [the code that is used by the site is open source](https://github.com/molly/web3-is-going-great) ([MIT License](https://github.com/molly/web3-is-going-great/blob/main/LICENSE)), there isn't a lot of documentation for their specific theme.

It does look like [their template](https://github.com/molly/static-timeline-generator) is better documented but without a solid understanding of HTML and nodejs/Javascript, it's going to be a challenge to get up and running.

That all being said, if you are taking classes, now is a good time to learn.  Follow the instructions outlined in the template repo and see if you can get it up and running.  You might be able to launch this in [Heroku](https://www.heroku.com/) for free.  Or, if you simply want to launch it on your own computer (but inaccessible to the outside world without changing router settings), you can install [nodejs](https://nodejs.org/en/) on your computer.

If your end-state is to have a website in general, Github offers [Github Pages](https://pages.github.com/) that's 100% free and they have [great templates](https://pages.github.com/themes/).  Github provides great documentation and allows you to do some light customization.  Doing github pages would be easier to learn as it's mostly markdown and very little coding required.
